pkgs/applications/science/misc/sasview/default.nix

```diff
@@ -1,70 +1,59 @@
-{ lib, fetchFromGitHub, gcc, python2 }:
+{ lib
+, python3
+, fetchFromGitHub
+, which
+, wrapQtAppsHook
+}:
 
-let
-  xhtml2pdf = import ./xhtml2pdf.nix {
-    inherit lib;
-    fetchPypi = python2.pkgs.fetchPypi;
-    buildPythonPackage = python2.pkgs.buildPythonPackage;
-    html5lib = python2.pkgs.html5lib;
-    httplib2 = python2.pkgs.httplib2;
-    nose = python2.pkgs.nose;
-    pillow = python2.pkgs.pillow;
-    pypdf2 = python2.pkgs.pypdf2;
-    reportlab = python2.pkgs.reportlab;
-  };
-
-in
-
-python2.pkgs.buildPythonApplication rec {
+python3.pkgs.buildPythonApplication rec {
   pname = "sasview";
-  version = "4.2.0";
+  version = "5.0.4";
 
-  checkInputs = with python2.pkgs; [
-    pytest
-    unittest-xml-reporting
-  ];
+  src = fetchFromGitHub {
+    owner = "SasView";
+    repo = "sasview";
+    rev = "v${version}";
+    hash = "sha256-TjcchqA6GCvkr59ZgDuGglan2RxLp+aMjJk28XhvoiY=";
+  };
 
-  checkPhase = ''
-    # fix the following error:
-    # imported module 'sas.sascalc.data_util.uncertainty' has this __file__ attribute:
-    #   /build/source/build/lib.linux-x86_64-2.7/sas/sascalc/data_util/uncertainty.py
-    # which is not the same as the test file we want to collect:
-    #   /build/source/dist/tmpbuild/sasview/sas/sascalc/data_util/uncertainty.py
-    rm -r dist/tmpbuild
+  doCheck = false; # TODO remove
```

Comment:
Was this meant to be left in?

Reply:
No. I knew I would forget and that's why I added the comment :D